src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/TabularEnergyMultipliers.hpp

```diff
@@ -0,0 +1,19 @@
+struct TabularEnergyMultipliers: 
```

Comment:
You may want to look at this one as well.

Reply:
Yeah, missed this one too. It's been fixed.